reView/cli.py

```diff
@@ -129,3 +135,259 @@ def unpack_characterizations(
+              default=None,
+              help=('Color map to use for the column. Refer to https://'
+                    'matplotlib.org/stable/tutorials/colors/colormaps.html'
+                    ' for valid options. If not specified, the viridis '
+                    'colormap will be applied.'))
+@click.option('--legend_title', '-T', required=False,
+              type=str,
+              default=None,
+              help=('Title to use for the map legend. '
+                    'If not provided, legend title will be the column name'))
+@click.option('--legend_breaks', '-B', required=False,
+              type=str,
+              default=None,
+              help=('Breaks to use for the map legend. Should be formatted '
+                    'like a list, e.g. : "[10, 50, 100, 150]". If not '
+                    'provided, a 5-class quantile classification will be used '
+                    'to derive the breaks.'))
+@click.option('--boundaries', '-b', required=False,
+              type=click.Path(exists=True, dir_okay=False, file_okay=True),
+              default=DEFAULT_BOUNDARIES,
+              # noqa: E126
+              help=('Path to vector dataset with the boundaries to map. '
+                    'Default is to use state boundaries for CONUS from '
+                    'Natural Earth (1:50m scale), which is suitable for CONUS '
+                    'supply curves. For other region, it is recommended to '
+                    'provide a more appropriate boundaries dataset.'
+                    ))
+@click.option('--dpi', '-d', required=False,
+              default=600,
+              type=click.IntRange(min=0),
+              help='Dots-per-inch (DPI) for output images. Default is 600.')
+def map_column(
+    supply_curve_csv, out_folder, column, colormap=None, legend_title=None,
+    legend_breaks=None, boundaries=DEFAULT_BOUNDARIES, dpi=600
+):
+    # pylint: disable=raise-missing-from
+    """
+    Generates a single map from an input supply curve for the specified column,
+    with basic options for formatting.
+    """
+
+    out_path = Path(out_folder)
+    out_path.mkdir(exist_ok=True, parents=False)
+
+    supply_curve_df = pd.read_csv(supply_curve_csv)
+    if column not in supply_curve_df.columns:
+        raise KeyError(
+            f"Column {column} could not be found in input supply curve."
+        )
+    supply_curve_gdf = gpd.GeoDataFrame(
+        supply_curve_df,
+        geometry=gpd.points_from_xy(
+            x=supply_curve_df['longitude'], y=supply_curve_df['latitude']
+        ),
+        crs="EPSG:4326"
+    )
+
+    boundaries_gdf = gpd.read_file(boundaries)
+    boundaries_singlepart_gdf = boundaries_gdf.explode(index_parts=True)
+
+    boundaries_dissolved = boundaries_gdf.unary_union
+    background_gdf = gpd.GeoDataFrame(
+        {"geometry": [boundaries_dissolved]},
+        crs=boundaries_gdf.crs
+    ).explode(index_parts=False)
+
+    map_extent = background_gdf.buffer(0.01).total_bounds
```

Comment:
`boundaries` is a user-input right? Do you need to cast it to a particular crs to make sure these buffer units are what you want? Or perhaps clearly state the crs you are expecting from the input